runtime/parachains/src/inclusion.rs

```diff
@@ -261,9 +267,25 @@ impl<T: Trait> Module<T> {
 		{
 			if pending_availability.availability_votes.count_ones() >= threshold {
 				<PendingAvailability<T>>::remove(&para_id);
+				let commitments = match <PendingAvailabilityCommitments>::take(&para_id) {
+					Some(commitments) => commitments,
+					None => {
+						debug::warn!(r#"
+						Inclusion::process_bitfields:
+							PendingAvailability and PendingAvailabilityCommitments
+							are out of sync, did someone mess with the storage?
+						"#);
+						continue;
+					}
+				};
```

Comment:
not sure if we want to remove `PendingAvailability` for para_id if `PendingAvailabilityCommitments` is not present
(storage is messed up)
i.e. should we put line 269 after the match?
probably doesn't matter, it's borked in any way

Reply:
It should be "ok" as-is, even though it's dealing with a degenerate state